src/hotspot/share/prims/jvmtiEnv.cpp

```diff
@@ -575,7 +575,7 @@ JvmtiEnv::SetEventNotificationMode(jvmtiEventMode mode, jvmtiEvent event_type, j
   if (event_type == JVMTI_EVENT_CLASS_FILE_LOAD_HOOK && enabled) {
     record_class_file_load_hook_enabled();
   }
-  JvmtiVTMSTransitionDisabler disabler(event_thread);
+  JvmtiVTMSTransitionDisabler disabler;
```

Comment:
An explanatory comment would have been good.

Reply:
The fix has been already integrated.
I'll add a comment when there is a chance.